scripts/screens/organizational_screens.py

```diff
@@ -916,6 +934,21 @@ def refresh_checkboxes(self):
                 container=self.checkboxes_text["container"],
                 tool_tip_text="Mild gore and blood will be allowed in the artwork displayed alongside patrols."
             )
+            
+            n += 1
+            # Emable discord rpc
+            if game.settings['discord']:
+                box_type = "#checked_checkbox"
+            else:
+                box_type = "#unchecked_checkbox"
+            self.checkboxes['discord'] = UIImageButton(
+                scale(pygame.Rect((x_value, n * y_spacing), (68, 68))),
+                "",
+                object_id=box_type,
+                container=self.checkboxes_text["container"],
+                tool_tip_text="Discord will show info about your clan."
```

Comment:
I suggest adding a line to the tooltip that make it clear you'll have to restart the game for this setting to take effect. In addition, maybe make it clear it will show details about your current game-state. Something like:
"Discord will display info about your game, including your clan's name. Game must be restarted changes to this setting to take effect. "

Reply:
the game doesnt have to be restarted though! the setting takes effect instantly